core/src/components/breadcrumb/breadcrumb.md.scss

```diff
@@ -0,0 +1,92 @@
+@import "./breadcrumb";
+@import "./breadcrumb.md.vars";
+
+// Breadcrumb
+// --------------------------------------------------
+
+:host {
+  --color: #{$breadcrumb-md-color};
+}
+
+:host(.breadcrumb-active) {
+  color: $breadcrumb-md-color-active;
+
+  font-weight: 500;
+}
+
+.breadcrumb-native {
+  @include padding(6px, 12px, 6px, 12px);
+}
+
+
+// Breadcrumb Separator
+// ------------------------------------------
+
+.breadcrumb-separator {
+  @include margin($breadcrumb-md-separator-margin-top, $breadcrumb-md-separator-margin-end, $breadcrumb-md-separator-margin-bottom, $breadcrumb-md-separator-margin-start);
+}
+
+
+// Breadcrumb: Hover
+// ------------------------------------------
+
+@media (any-hover: hover) {
+  :host(:hover) {
+    color: $breadcrumb-md-color-active;
+  }
+
+  :host(.in-breadcrumbs-color:hover),
+  :host(.ion-color:hover) {
+    color: #{current-color(shade)};
+  }
+}
+
+
+// Breadcrumb: Focused
+// ------------------------------------------
+
+:host(.ion-focused) .breadcrumb-native {
+  @include border-radius(4px);
+
+  box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.2), 0px 2px 8px rgba(0, 0, 0, 0.12);
+}
+
+
+// Breadcrumb Slotted Icons
+// ------------------------------------------
+
+::slotted(ion-icon) {
+  color: #7d8894;
```

Comment:
Along with the other hardcoded hex values, can we update this to work with the `color` property?